.github/workflows/build.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
       # Push events on the main branch
       - main
+      - release/**
```

Comment:
> we will no longer have to update .release/ci.hcl whenever we need to trigger CRT for a new release branch.

This sounds awesome 🎉 🥳 

I'm thinking about all the "older" release branches. Quick questions. Not just for this `build` wf but for the `ci` wfs, do we want to block releases older than `1.10` or `1.9` ?

Reply:
Yeah, you bring up a good question that I thought about a bit. Basically, I thought about writing a more strict pattern to match against that involved only matching our active release series, but this option has a couple downsides:
1. it's less readable
2. it still means we need to update whenever we have a new release series which defeats some of the value of having a wildcard at all
3. it doesn't really buy us much because these old release branches aren't being updated

So, with those bits in mind, I just went for the simpler, always up-to-date, less restrictive option that you're seeing here. It also means that if someone wants to create a hotfix branch like `release/1.11.3-myfixes` they can and get build artifacts right away. 

I'm open to discussing more, but those are my thoughts so far. Thanks for bringing it up.